dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/server.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+    <include location="../fatTestPorts.xml"/>
+    
+	<featureManager>
+	    <feature>componenttest-1.0</feature>
+		<feature>osgiconsole-1.0</feature>
+		<feature>localConnector-1.0</feature>
+		<feature>appSecurity-2.0</feature>
+		<feature>cdi-1.2</feature>
```

Comment:
Emily changed the FaultTolerance bucket to run against cdi-2.0 by default and then just selected tests with cdi-1.2. You might wish to follow the same pattern.